ts/Extensions/DataGrouping.ts

```diff
@@ -169,7 +174,7 @@ declare global {
         let defaultDataGroupingUnits: Array<[string, (Array<number>|null)]>;
         type DataGroupingApproximationValue = (
             'average'|'averages'|'ohlc'|'open'|'high'|'low'|'close'|'sum'|
-            'windbarb'|'ichimoku-averages'
+            'windbarb'|'ichimoku-averages'|'hlc'
```

Comment:
In `@typedef` the `ohlc` and `hlc` is missing.

Additionally, to the `approximation` API documentation you should add what is the default type of approximation for the HLC series.